### DISCUSSION TITLE
Could not find artifact org.apache.kafka:kafka-clients:jar:2.9.13

### DISCUSSION THREAD

**Discussion by anudhij** [spring-projects/spring-kafka#2980] [Q&A] [answered]
Hey all, I am using Spring Boot 2.7.2, and while adding maven dependency:

        <dependency>
            <groupId>org.springframework.kafka</groupId>
            <artifactId>spring-kafka</artifactId>
            <version>2.9.13</version>
        </dependency>

I am getting below error: 

 The following artifacts could not be resolved: org.apache.kafka:kafka-clients:jar:2.9.13 (absent): Could not find artifact org.apache.kafka:kafka-clients:jar:2.9.13 in central (https://repo.maven.apache.org/maven2)

From the list I can see there are no 2.9.x versions present: https://repo.maven.apache.org/maven2/org/apache/kafka/kafka-clients/
As per the dicumentation, it says: Spring Boot 2.7 users should use 2.9.x (Boot dependency management will use the 2.8.x version). 
https://spring.io/projects/spring-kafka/

Could you help in identifying the root cause?

**Comment by gpando** [comment, marked as answer]
https://docs.spring.io/spring-kafka/docs/2.9.13/reference/html/#update-deps


```
<properties>
    <kafka.version>3.6.0</kafka.version>
    <spring-kafka.version>2.9.13</spring-kafka.version>
</properties>

<dependency>
    <groupId>org.springframework.kafka</groupId>
    <artifactId>spring-kafka</artifactId>
</dependency>
```
